lib/promscrape/scrapework.go

```diff
@@ -833,6 +839,7 @@ func isAutoMetric(s string) bool {
 func (sw *scrapeWork) addAutoMetrics(am *autoMetrics, wc *writeRequestCtx, timestamp int64) {
 	sw.addAutoTimeseries(wc, "up", float64(am.up), timestamp)
 	sw.addAutoTimeseries(wc, "scrape_duration_seconds", am.scrapeDurationSeconds, timestamp)
+	sw.addAutoTimeseries(wc, "scrape_response_size_bytes", am.scrapeResponseSize, timestamp)
```

Comment:
I'm not sure where is this metric will be used, as we already have `vm_promscrape_scrape_response_size_bytes_bucket` to record the response size.

Reply:
adding `scrape_response_size_bytes` to auto metrics allows to track response size for a specific scrape target and to understand which values should be set for scrape size limits and for which targets

Reply:
What about adding this info to the `/targets` page of vmagent ui, like `scrapeDuration` or `samplesScraped`?
<img width="2505" alt="image" src="https://github.com/VictoriaMetrics/VictoriaMetrics/assets/39937150/7319a68d-2d39-4bf1-b2fd-0f2ee1371029">

Reply:
done